pymc/distributions/distribution.py

```diff
@@ -460,19 +452,21 @@ def __new__(
         if not isinstance(name, string_types):
             raise TypeError(f"Name needs to be a string but got: {name}")
 
-        # Create the RV and process dims and observed to determine
-        # a shape by which the created RV may need to be resized.
-        rv_out, dims, observed, resize_shape = _make_rv_and_resize_shape(
+        dims = convert_dims(dims)
+        if observed is not None:
+            observed = convert_observed_data(observed)
+
+        # Create the RV, without taking into consideration `dims`
+        rv_out, resize_shape_from_dims = _make_rv_and_resize_shape_from_dims(
             cls=cls, dims=dims, model=model, observed=observed, args=args, **kwargs
         )
 
-        if resize_shape:
-            # A batch size was specified through `dims`, or implied by `observed`.
-            rv_out = cls.change_size(
-                rv=rv_out,
-                new_size=resize_shape,
-                expand=True,
+        # Resize variable based on `dims` information
+        if resize_shape_from_dims:
```

Comment:
Why is this not covered?

Reply:
We didn't have a case with SymbolicDistributions. I am removing this in the follow up PR I already opened anyway, so I didn't want to add nearly duplicated tests just for this.

Reply:
Ah I didn't notice that this was in `SymbolicDistribution`